openstack/cronus/Chart.yaml

```diff
@@ -1,14 +1,14 @@
 apiVersion: v2
 name: cronus
 description: proxying hyperscaler statless services
-version: 0.1.13
+version: 0.1.14
 dependencies:
   - name: rabbitmq
     repository: oci://keppel.eu-de-1.cloud.sap/ccloud-helm
     version: 0.6.12
   - name: rabbitmq
     repository: oci://keppel.eu-de-1.cloud.sap/ccloud-helm
-    version: 0.6.12
+    version: 0.7.2
```

Comment:
It looks like only 1 rabbitmq was upgraded. Is this expected?

Reply:
Unexpected. Added version bump for the second one.